kivy/uix/behaviors/focus.py

```diff
@@ -237,6 +237,15 @@ def _get_keyboard(self):
         2.0.0.
     '''
 
+    keyboard_suggestions = BooleanProperty(True)
+    '''If True provides auto suggestions on top of keyboard.
+    This will only work if :attr:`input_type` is set to `text`, `url`, `mail` or
+    `address`.
+    .. versionadded:: 2.0.0
```

Comment:
```suggestion

    .. versionadded:: 2.0.0

```

Reply:
Sounds good. I will do it today. Thanks @matham for getting back to me quickly.

Reply:
Oh, and I just noticed, it should be 2.1.0, not 2.0.0.

Reply:
@matham regarding the request_keyboard in window_pygame.py, what do you want to me change regarding it? The android.show_keyboard looks to handle keyboard suggestions and input type already.

Reply:
Just add the new keyboard to `request_keyboard` here https://github.com/kivy/kivy/blob/master/kivy/core/window/window_pygame.py#L435. It doesn't have to do anything with, just pass it on to super I guess. Because I think right now it would crash with the unexpected keyword.

Reply:
fixed it.